packages/soupault/soupault.4.3.1/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "2.0"
+synopsis: "Static website generator based on HTML rewriting"
+description: """\
+A website generator that works with page element tree rather than text
+and allows you to manipulate pages and retrieve metadata from
+existing HTML using arbitrary CSS selectors.
+
+With soupault you can:
+
+* Generate ToC and footnotes.
+* Insert file content or an HTML snippet in any element.
+* Preprocess element content with external programs (e.g. run `<pre>` tags through a highlighter)
+* Extract page metadata (think microformats) and render it using a Jingoo template or an external script.
+* Export extracted metadata to JSON.
+
+Soupault is extensible with Lua (2.5) plugins and provides an API for element tree manipulation,
+similar to web browsers.
+
+The website generator mode is optional, you can use it as post-processor for existing sites."""
+maintainer: "Daniil Baturin <daniil+opam@baturin.org>"
+authors: "Daniil Baturin <daniil+soupault@baturin.org>"
+license: "MIT"
+homepage: "https://www.soupault.app"
+bug-reports: "https://github.com/PataphysicalSociety/soupault/issues"
+depends: [
+  "ocaml" {>= "4.08"}
```

Comment:
```suggestion
  "ocaml" {>= "4.13"}
```
Due to
```
#=== ERROR while compiling soupault.4.3.1 =====================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://codeberg.org/PataphysicalSociety/soupault/archive/4.3.1.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/soupault.4.3.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p soupault -j 71
# exit-code            1
# env-file             ~/.opam/log/soupault-7-d3568d.env
# output-file          ~/.opam/log/soupault-7-d3568d.out
### output ###
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.soupault.eobjs/byte -I /home/opam/.opam/4.08/lib/base64 -I /home/opam/.opam/4.08/lib/bigarray-compat -I /home/opam/.opam/4.08/lib/camomile -I /home/opam/.opam/4.08/lib/camomile/default_config -I /home/opam/.opam/4.08/lib/camomile/dyn -I /home/opam/.opam/4.08/lib/camomile/lib_default -I /home/opam/.opam/4.08/lib/camomile/library -I /home/opam/.opam/4.08/lib/containers -I /home/opam/.opam/4.08/lib/containers/monomorphic -I /home/opam/.opam/4.08/lib/cstruct -I /home/opam/.opam/4.08/lib/ctypes -I /home/opam/.opam/4.08/lib/either -I /home/opam/.opam/4.08/lib/ezjsonm -I /home/opam/.opam/4.08/lib/fileutils -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/hex -I /home/opam/.opam/4.08/lib/integers -I /home/opam/.opam/4.08/lib/jingoo -I /home/opam/.opam/4.08/lib/jsonm -I /home/opam/.opam/4.08/lib/lambdasoup -I /home/opam/.opam/4.08/lib/logs -I /home/opam/.opam/4.08/lib/lua-ml -I /home/opam/.opam/4.08/lib/markup -I /home/opam/.opam/4.08/lib/menhirLib -I /home/opam/.opam/4.08/lib/odate -I /home/opam/.opam/4.08/lib/otoml -I /home/opam/.opam/4.08/lib/ppx_deriving/runtime -I /home/opam/.opam/4.08/lib/re -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/sexplib0 -I /home/opam/.opam/4.08/lib/spelll -I /home/opam/.opam/4.08/lib/stdlib-shims -I /home/opam/.opam/4.08/lib/tsort -I /home/opam/.opam/4.08/lib/uchar -I /home/opam/.opam/4.08/lib/uucp -I /home/opam/.opam/4.08/lib/uutf -I /home/opam/.opam/4.08/lib/yaml -I /home/opam/.opam/4.08/lib/yaml/bindings -I /home/opam/.opam/4.08/lib/yaml/bindings/types -I /home/opam/.opam/4.08/lib/yaml/c -I /home/opam/.opam/4.08/lib/yaml/ffi -I /home/opam/.opam/4.08/lib/yaml/types -no-alias-deps -open Dune__exe -o src/.soupault.eobjs/byte/dune__exe__Plugin_api.cmo -c -impl src/plugin_api.ml)
# File "src/plugin_api.ml", line 827, characters 82-100:
# 827 |        "starts_with", V.efunc (V.string **-> V.string **->> V.bool) (fun s pat -> String.starts_with ~prefix:pat s);
#                                                                                         ^^^^^^^^^^^^^^^^^^
# Error: Unbound value String.starts_with
```

Reply:
Oh, indeed! Sorry, I forgot to update the constraint. Thanks for fixing it.